rasa/core/actions/forms.py

```diff
@@ -455,7 +455,9 @@ async def validate(
         )
 
         some_slots_were_validated = any(
-            isinstance(event, SlotSet) for event in validation_events
+            isinstance(event, SlotSet)
+            for event in validation_events
+            if isinstance(event, SlotSet) and not event.key == REQUESTED_SLOT
```

Comment:
could you please add an explanatory comment.

Reply:
done